src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -11,19 +11,29 @@
 
 namespace Symfony\Component\Serializer\Tests\Encoder;
 
+use DateTime;
```

Comment:
For classes in the global namespace, we don't import them (use `\DateTime` directly).